Katana/Store.swift

```diff
@@ -289,12 +313,14 @@ open class Store<S: State, D: SideEffectDependencyContainer>: PartialStore<S> {
 
    - parameter interceptors: a list of interceptors that are executed every time something is dispatched
    - parameter stateInitializer: a closure invoked to define the first state's value
+   - parameter dependenciesInitializer: a closure invoked to instantiate the dependencies
    - parameter configuration: the configuration needed by the store to start properly
    - returns: An instance of store
    */
   public init(
     interceptors: [StoreInterceptor],
     stateInitializer: @escaping StateInitializer<S>,
+    dependenciesInitializer: @escaping DependenciesInitializer,
```

Comment:
```suggestion
    dependenciesInitializer: @escaping DependenciesInitializer = { dispatch, getState in D(dispatch: dispatch, getState: getState) },
```

so that you can avoid the double init?

Reply:
As discussed, keeping it like this to avoid to be breaking and for consistency with the other initializers